yaqd_wright/ndinterp.avpr

```diff
@@ -457,6 +458,7 @@
         }
     },
     "traits": [
+        "has-dependents",
         "has-limits",
         "has-position",
         "is-daemon",
```

Comment:
how is it that `compose` generated traits for has-position when that wasn't part of the .toml indication?   And why is is-homeable missing when it was part of the .toml?

Reply:
has-limits inherits has-position, and yaq-traits knows these dependencies

Reply:
Line 465 has is-homeable